packages/dd-trace/src/appsec/blocking.js

```diff
@@ -2,19 +2,19 @@
 
 const fs = require('fs')
 let templateHtml, templateJson
-function block (req, res, topSpan, abortController) {
+function block ({ req, res, topSpan, abortController }) {
   let type
-  let body
+  let bodyContents
```

Comment:
why this rename?

Reply:
Because, originally, we thought that the user could customize the body response so when I refactored the function for the first time I added a parameter in order to pass the body like this:
`function block ({ req, res, body, topSpan, abortController })`
Hence I renamed the internal variable. Later when we decided to remove body from function's parameters I did not undo the variable name change.